crates/sdk/src/tx.rs

```diff
@@ -3548,10 +3548,15 @@ pub async fn build_update_account(
         let threshold = *threshold;
 
         let invalid_threshold = threshold.is_zero();
-        let invalid_threshold_updated = !public_keys.is_empty() && public_keys.len() < threshold as usize;
-        let invalid_threshold_current = public_keys.is_empty() && account.get_all_public_keys().len() < threshold as usize;
-
-        if invalid_threshold || invalid_threshold_updated || invalid_threshold_current {
+        let invalid_threshold_updated =
```

Comment:
was this not fmt'ed in #3566? ci didn't complain

Reply:
very confused about this too :S

Reply:
ah, it was running `make fmt` instead of `make fmt-check` (https://github.com/anoma/namada/pull/3583)

Reply:
rip 🪦 nice catch!